tests/components/sharePanel-test.js

```diff
@@ -70,11 +69,76 @@ describe('SharePanel', function () {
         TestUtils.Simulate.click(twitter);
         TestUtils.Simulate.click(facebook);
         TestUtils.Simulate.click(googlePlus);
+        OO={};
```

Comment:
Why do we need to set OO to an empty object here?

Reply:
handleEmailClick: function(event){}

I have added code to check ios and safari browser flags handleEmailClick() function.
That is the reason I added this code. Without these those flags undefined inside the function while running unit testing.

Reply:
@aeng7 It looks like in the current test environment none of the required files create OO so he is creating it here. Do we have a better way or should he add a require in?